extensions/adobe/experience/campaign/feedbackevent.schema.json

```diff
@@ -0,0 +1,244 @@
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/campaign/feedbackevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Delivery Feedback Events",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Contains all delivery event fields.",
+  "definitions": {
+    "feedbackevent": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/feedbackType": {
+          "title": "Feedback Type",
+          "type": "string",
+          "description": "The type of message event."
+        },
+        "https://ns.adobe.com/experience/campaign/retryCount": {
+          "title": "Retry Count",
+          "type": "integer",
+          "description": "Number of failed attempts before the current event."
+        },
+        "https://ns.adobe.com/experience/campaign/delivery": {
+          "title": "The campaign activity originating a message to a recipient.",
+          "type": "object",
+          "properties": {
+            "xdm:id": {
+              "title": "Delivery ID",
+              "type": "string",
+              "description": "The campaign activity originating this message."
+            },
+            "xdm:campaignId": {
+              "title": "Campaign Id",
+              "type": "string",
+              "description": "Campaign of which this delivery was a part."
+            },
+            "xdm:from": {
+              "title": "From",
+              "type": "string",
+              "description": "Address used as a from/sender/caller address. Depending on the communication channel, the `address` has a different format.\n\n  * `email`: an email address.\n  * `textMessage` or `phone`: a shortcode or phone number."
+            },
+            "xdm:type": {
+              "title": "Delivery Type",
+              "type": "string",
+              "description": "Mode of delivery for the messages sent."
+            },
+            "xdm:label": {
+              "title": "Delivery Label",
+              "type": "string",
+              "description": "A human-friendly name of the campaign activity which is originating this message."
+            },
+            "xdm:name": {
+              "title": "Delivery Internal Name",
+              "type": "string",
+              "description": "A human-friendly identifier of the campaign activity which is originating this message."
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/message": {
+          "title": "A single message sent to a recipient.",
+          "type": "object",
+          "properties": {
+            "xdm:id": {
+              "title": "Message Identifier",
+              "type": "string",
+              "description": "Identifier of the message."
+            },
+            "xdm:subject": {
```

Comment:
the "subject" is a very open and wider concept. It could be the study of anyThing. In this case, it is semantically "subject of the email message" sent to recipients. Use xdm:emailSubject to make it more meaningful or use Subject under your namespace.

Reply:
The delivery feedback events that we are defining could be for email/push/sms. Any field specific to these channels will go in their channel block. All other fields which are common to all of these channels are defined in rest of the blocks. So, subject here would also mean subject of the push notification or sms.